### ISSUE TITLE
FIXME: Conflict with gog Mac version as it is named RimWorld.app not RimWorldMac.app

### ISSUE THREAD

**Issue by Ant-V** [RimSort/RimSort#171]
**Describe the bug**
It is not finding the game directory, since it is named RimWorld.app instead of RimWorldMac.app

**To Reproduce**
On any new installation from gog on macOS it cannot autodetect the `game_folder` or the `local_folder`.
Setting the paths by editing `settings.json` to 
```
    "game_folder": "/Applications/" **OR** "/Applications/RimWorld.app"
    "local_folder": "/Applications/RimWorld.app/Mods",
```

causes a pop up with the error:
> ***Issue Getting Game Version***
> RimSort is unable to get the game version at the expected path: [/Applications/RimWorldMac.app/Version.txt]. Is your game path [/Applications/RimWorldMac.app] set correctly? There should be a Version.txt file in the game install directory.

Followed by a crash once `OK` is clicked

**Expected behavior**
Simple solutions may include: 
* remove default name appending in path
* allow selection of folders or .app when selecting path
* check for both naming schemes

**Please complete the following information:**
 - OS: macOS 14.0 
 - RimSort version: alpha v1.0.6.2-hf
 - Please provide a `RimSort.log`: none created
 - Are you running from Python interpreter or from a compiled RimSort release? RimSort release.

**Comment by oceancabbage** [comment]
Looking into this now. Thinking of going with your suggestion 2 and 3: adding autodetect support for people who play the GoG version (I'm assuming the default path is how you have it, in `Applications`; and, allowing manual selection of folders or `.app`.

Not 100% sure what you mean by "remove default name appending in path", do you want to clarify on that?

**Comment by Ant-V** [comment]
Sorry, I may have not been clear enough, because I had to post it twice because of a timeout error on githubs part.

When I enter a custom path directly in the settings, it appends the "RimWorldMac.app" in the path, so for example when I entered `"/Applications/RimWorld.app"` the error was `RimSort is unable to get the game version at the expected path: [/Applications/RimWorld.app/RimWorldMac.app/Version.txt]  Is your game path [/Applications/RimWorld.app/RimWorldMac.app] set correctly?`, which I (not so descriptively) would described as appending the default name in the path.

Maybe checking if Version.txt exists before appending the default `RimWorldMac.app` name in the path could also be a solution.

I could try a fix and posting a pull request, but sadly python is a language I almost never use.
If I have time later today however I may give it a go.
